src/main.rs

```diff
@@ -48,17 +48,19 @@ use std::path::PathBuf;
 #[macro_export]
 macro_rules! print_error {
     ($($arg:tt)*) => {
-        use std::io::Write;
-
-        let stderr = std::io::stderr();
-
         {
-            let mut handle = stderr.lock();
-            // We can write on stderr, so we simply ignore the error and don't print
-            // and stop with success.
-            let res = handle.write_all(std::format!($($arg)*).as_bytes());
-            if res.is_err() {
-                std::process::exit(0);
+            use std::io::Write;
+
+            let stderr = std::io::stderr();
+
+            {
+                let mut handle = stderr.lock();
+                // We can write on stderr, so we simply ignore the error and don't print
+                // and stop with success.
+                let res = handle.write_all(std::format!($($arg)*).as_bytes());
```

Comment:
We could use a newline at the end.

Reply:
I updated this macro to use a format like `lsd: Message.\n\n`

`lsd` and two new lines are inspired by some previous error output, this shows the lsd name and adds a new line to separate the err msg and ls output.